docs/sources/reference/api/registry_api.md

```diff
@@ -536,6 +536,8 @@ Search the Index given a search term. It accepts
 Query Parameters:
 
 - **q** – what you want to search for
+- **n** - number of results you want returned per page (default: 25)
```

Comment:
Ahh sorry, forgot to respond

Good catch, I guess I *didnt* actually account for edge case of n=0. I'll update that. We decided that querying for all the results was costly and inefficient so that will no longer be an option. At most, users will be able to search for 100 at a time.

If page=xxxx is out of range it will default to the last page returned. If the page isn't a number it will default to the first page returned.

I'll add these changes and update this pr once the changes are submitted